### DISCUSSION TITLE
CC: Tweaked 1.96.0

### DISCUSSION THREAD

**Discussion by SquidDev** [cc-tweaked/CC-Tweaked#795] [Announcements]
It's been a while since the last CC: Tweaked release (and even longer since the last 1.x.0 one). But well, 1.96.0 is out, and while there's not anything big, there's definitely some nice changes:

 - The generic peripheral system is now exposed on the public API, which means mod authors should be able to use it too.
 -  Inventories now also contain a `getLimit` method to get the maximum number of items in the stack.
 - @Lupus590 and @MCJack123 have both contributed a couple of small improvements to `cc.expect`.
 - Lots of bug fixes and improvements to documentation.

As always, many thanks to everyone who has contributed over the last few months!

**Comment by Lupus590** [comment]
I just wanted to make [this](https://github.com/Lupus590-CC/CC-TreeFarm/blob/master/treeFarm/libs/utils/argValidationUtils.lua) a bit more obsolete.

**Comment by Merith-TK** [comment]
This will be interesting to port to fabric.

**Comment by MCJack123** [comment]
Well, maybe this is the time for me to stop procrastinating on fixing `xpcall` resumability... or maybe I'll patch out the tests and do it later ¯\\\_(ツ)\_/¯ Other than that, updating CraftOS-PC to 1.96.0 shouldn't be too bad since it's mostly Lua changes.
